src/io/parquet/read/deserialize/binary/utils.rs

```diff
@@ -25,6 +25,9 @@ impl<O: Offset> Offsets<O> {
 }
 
 impl<O: Offset> Pushable<O> for Offsets<O> {
+    fn reserve(&mut self, additional: usize) {
```

Comment:
```suggestion
    #[inline]
    fn reserve(&mut self, additional: usize) {
```

some reserves are used inside loops